File: sdks/python/bolt_prediction_sdk/api/system.py
"""
System API 客户端
"""

from typing import Optional, Dict, Any, List, AsyncIterator
import json

from ..core.client import BaseAPIClient
from ..core.pagination import CursorPaginator
from ..models import *


class SystemClient(BaseAPIClient):
    """System API 客户端"""

    async def health_check_health_get(
        self
) -> HealthResponse:
        """
        健康检查（公开免鉴权）
        """

        response = await self._request(
            method="GET",
            path=f"/api/v1/health",
        )

        return response

File: sdks/typescript/src/api/system.ts
/** System API 客户端 */

import { BaseAPIClient } from "../core/client";
import { CursorPaginator } from "../core/pagination";
import * as Models from "../models";

export class SystemClient extends BaseAPIClient {

  /**
   * 健康检查（公开免鉴权）
   */
  async healthCheckHealthGet(
  ): Promise<Models.HealthResponse> {
    const params: Record<string, any> = {};

    return this._request(
      "GET",
      `/api/v1/health`,
      params,
      undefined
    );
  }
}